src/Paystack.php

```diff
@@ -107,7 +107,23 @@ public function makePaymentRequest($data = null)
     }
 
     /**
-     * Get the authorization url from the callback response.
+     * Create the authorization url from the callback response.
+     *
+     * @param null $data
+     *
+     * @return Paystack
+     */
+    public function createAuthorizationUrl($data = null)
+    {
+        $this->makePaymentRequest($data);
```

Comment:
For now, can we let this just call and return `getAuthorizationUrl`, since both basically are the samething like so:
```php
public function createAuthorizationUrl($data = null)
{
     return $this->getAuthorizationUrl($data);
}
```

Reply:
Yes, this would work.